src/rqt_launch/node_proxy.py

```diff
@@ -71,12 +71,12 @@ def __init__(self, run_id, master_uri, config):
     # LocalProcess.is_alive() does not do what you would expect
     def is_running(self):
         rospy.logdebug('BEFORE started={}, stopped={} alive={}'.format(
-                                                     self._process.started,
-                                                     self._process.stopped,
-                                                     self._process.is_alive()))
+            self._process.started,
```

Comment:
Align with str.format

Reply:
I generally don't like doing that because it makes the indentation dependent on the string being formatted and every change to the string then requires we re-indent for a bunch of lines